src/App.js

```diff
@@ -1,5 +1,40 @@
-function App() {
-  return <div className="App"></div>
+import { useState, useEffect } from 'react'
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+export default function App() {
+  const [tracks, setTracks] = useState([])
+
+  useEffect(() => getTracks(), [])
+
+  function getTracks() {
+    fetch('http://wandergold.local/track')
+      .then((res) => res.json())
+      .then((data) => setTracks(data))
+      .catch((error) => console.error('Error:', error))
+  }
```

Comment:
You could write the fetch as a service function.

Reply:
Yes, that will definitely be done.